_author_instructions/01_policies.md

```diff
@@ -333,6 +337,10 @@ Authors are responsible for obtaining all needed permissions to include previous
 If an article is an adaptation of a previously published article, it must be noted in the submission cover letter and major changes noted.
 Evaluating whether such changes constitute a significant revision will be part of the review process.
 
+The U.S. National Science Foundation definition of plagiarism as "the appropriation of another person’s ideas, processes, results, or words without giving appropriate credit" (45 Code of Federal Regulations, Section 689.1). Authors should not engage in plagiarism, which operationally means verbatim or near-verbatim copying, or even very close paraphrasing, of text or results from another’s work. 
+
+Authors should not engage generally engage self-plagiarism, i.e. close replication of the authors' own previously published without acknowledgement of the source. Material quoted verbatim from the authors' own previously published work may be acceptable, as long as the overall work clearly notes that the work is partially based on identified previous work, and the broad outlines of the differences and similarities of the previous work are noted. This use of previous work only applies to material that could be reasonably considered review or pedagogcial; in no context should previously published results or methodology be depicted as new.  We emphasize again that authors themselves are responsible for obtaining all needed permissions to include previously published material, including figures.
```

Comment:
It seems like the exception to this paragraph is if someone has published previously under a permissive license that allows them to adapt their material for later reuse (e.g. with Ann. Rev., or under a CC-BY license elsewhere, or...) in which case they would simply need to acknowledge/cite the prior work and explain how they've adapted it, probably. 

Your statement gets close to saying something like this, but this part is confusing: "Material quoted verbatim from the authors' own previously published work may be acceptable, as long as the overall work clearly notes that the work is partially based on identified previous work, and the broad outlines of the differences and similarities of the previous work are noted." How can it be quoted verbatim if it's also similar and different? 

I think the correct policy should be something like:
- Any quotes from prior work are given as quotes, with proper citation/acknowledgement (and permissions), even if that prior work is from the authors
- Any adaptation from prior work (by the authors) is allowed only of the prior license permits, and the original source should be cited/acknowledged and the adaptation explained

Reply:
OK, made changes to that effect.